lib/bundle/mac_app_store_installer.rb

```diff
@@ -0,0 +1,21 @@
+module Bundle
+  class MacAppStoreInstaller
+    def self.install(name, id)
```

Comment:
Yeh, sorry, I mean I know how to do so but I'm just unsure if we want to. I guess if someone is running `brew bundle` on a `Brewfile` that contains `mas` entries: we probably do.